CSC8503/GameSceneManager.cpp

```diff
@@ -69,6 +81,11 @@ void GameSceneManager::CreateLevel() {
 	mLevelManager->LoadLevel(0, 0);
 }
 
+void GameSceneManager::ClearLevel() {
+	mLevelManager->GetGameWorld()->ClearAndErase();
+	mLevelManager->GetPhysics()->Clear();
+}
```

Comment:
I have a method for this in LevelManager already, mLevelManager->ResetLevel() will do this